apollo-router/tests/integration/fixtures/broken_coprocessor.router.yaml

```diff
@@ -0,0 +1,6 @@
+# This coprocessor doesn't point to anything
+coprocessor:
+  url: "http://localhost:10000"
```

Comment:
Would `39000` (an impossible port), still recreate this?  I'm worried that `10000` may actually magically appear at some point in this repository. 🤣 
```suggestion
  url: "http://localhost:10000"
```

Reply:
We can use a domain that ends in `.example` or `.invalid` instead of localhost, it's guaranteed to not be routable

Reply:
[56112b1](https://github.com/apollographql/router/pull/5159/commits/56112b12e457d7a185049fa1110a53e4c535beec)

Reply:
[b15e4f2](https://github.com/apollographql/router/pull/5159/commits/b15e4f2e9dba6beb82f58a9aab1d15c68b86faf3)